darts/ad/utils.py

```diff
@@ -422,105 +425,92 @@ def show_anomalies_from_scores(
 
         nbr_plots = nbr_plots + len(set(window))
 
-    fig, axs = plt.subplots(
-        nbr_plots,
-        figsize=(8, 4 + 2 * (nbr_plots - 1)),
-        sharex=True,
-        gridspec_kw={"height_ratios": [2] + [1] * (nbr_plots - 1)},
-        squeeze=False,
-    )
-
-    index_ax = 0
-
-    _plot_series(series=series, ax_id=axs[index_ax][0], linewidth=0.5, label_name="")
-
-    if pred_series is not None:
-        _plot_series(
-            series=pred_series,
-            ax_id=axs[index_ax][0],
-            linewidth=0.5,
-            label_name="model output",
+    if multivariate_plot:
+        series = _check_input(
+            series,
+            name="series",
+            check_multivariate=True,
+        )[0]
+        series_width = series.n_components
+
+        if pred_series is not None:
+            pred_series = _check_input(
+                pred_series,
+                name="pred_series",
+                width_expected=series.width,
+                check_multivariate=True,
+            )[0]
+
+        if anomalies is not None:
+            anomalies = _check_input(
```

Comment:
If this check is performed, we should make sure this series contains binary values